Dockerfile

```diff
@@ -38,7 +38,8 @@ FROM scratch AS vendor-update
 COPY --from=vendored /out /
 
 FROM vendored AS vendor-validate
-RUN --mount=type=bind,target=.,rw <<EOT
+RUN --mount=target=/context \
+    --mount=target=.,type=tmpfs <<EOT
```

Comment:
Wondering if we should change all the `.`'s with an absolute paths (I think it's `/src` in this case?). Currently I have to read "up" in the Dockerfile to understand what `.` refers to.

Do we know why a `tmpfs` is needed for this? Is this an issue with `overlayfs` not supporting some attributes, or what's the cause?

Reply:
Or is there no content at all (as there's no `COPY`) and that was the issue?

Reply:
> Or is there no content at all (as there's no `COPY`) and that was the issue?

Yes that was the issue